R/D1Client.R

```diff
@@ -611,7 +611,7 @@ setMethod("getDataPackage", "D1Client", function(x, identifier, lazyLoad=FALSE,
   # Download the resource map, parse it and load the relationships into the DataPackage
   # Currently we only use the first resource map
   if(!quiet) cat(sprintf("Getting resource map with id: %s\n", dpkg@resmapId))
-  resMapObj <- getDataObject(x, identifier=resmapId, lazyLoad=FALSE, checksumAlgorithm=checksumAlgorithm)
+  resMapObj <- getDataObject(x, identifier=resmapId, lazyLoad=FALSE, limit=limit, checksumAlgorithm=checksumAlgorithm)
```

Comment:
Shouldn't this be `limit='1TB'` if you want to set the limit to a high value for resource maps rather than using the value the user provided?

Reply:
Thanks Matt - totally got it mixed up when I was editing it late a night. Thank you for catching it!